mojaloop/values.yaml

```diff
@@ -7294,8 +9926,26 @@ mojaloop-bulk:
         # this can be set if the dependency chart for kafka is disabled. If 'kafka_host' is commented out, then the name of the dependency chart will be used.
         kafka_host: *KAFKA_HOST
         kafka_port: *KAFKA_PORT
+
+        ## MongoDB Configuration for Object Store
+        mongo_host: *OBJSTORE_MONGO_HOST
+        mongo_port: *OBJSTORE_MONGO_PORT
+        mongo_user: *OBJSTORE_MONGO_USER
+        ## Secret-Management
+        ### Set this if you are using a clear password configured in the config section
+        mongo_password: *OBJSTORE_MONGO_PASSWORD
```

Comment:
```suggestion
        mongo_password: ''
```

Reply:
@elnyry-sam-k this is arelady set on the top-level config --> https://github.com/mojaloop/helm/blob/feat/%233082-utilise-externalised-secrets-for-user-password-management/mojaloop/values.yaml#L2